src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
```

Comment:
[CODE STYLE] - Use the `-=` operator to subtract from the existing value. Replace `return valueResult - value;` with `valueResult -= value; return valueResult;`.